library.wdl

```diff
@@ -49,37 +50,41 @@ workflow Library {
             starIndexDir = starIndexDir
     }
 
+    if (variantCalling) {
     # Preprocess BAM for variant calling
-    call picard.MarkDuplicates as markDuplicates {
-        input:
-            inputBams = [starAlignment.bamFile.file],
-            inputBamIndexes = [starAlignment.bamFile.index],
-            outputBamPath = outputDir + "/" + sampleId + "-" + libraryId + ".markdup.bam",
-            metricsPath = outputDir + "/" + sampleId + "-" + libraryId + ".markdup.metrics"
+        call picard.MarkDuplicates as markDuplicates {
```

Comment:
I suppose this is fine, but MarkDuplicates could be kept outside of the if. It shouldn't influence counting result.